src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -50,13 +51,15 @@ public class HubConnectionContext
         /// <param name="keepAliveInterval">The keep alive interval. If no messages are sent by the server in this interval, a Ping message will be sent.</param>
         /// <param name="loggerFactory">The logger factory.</param>
         /// <param name="clientTimeoutInterval">Clients we haven't heard from in this interval are assumed to have disconnected.</param>
-        public HubConnectionContext(ConnectionContext connectionContext, TimeSpan keepAliveInterval, ILoggerFactory loggerFactory, TimeSpan clientTimeoutInterval)
+        /// <param name="streamBufferCapacity">The buffer size for client upload streams</param>
+        public HubConnectionContext(ConnectionContext connectionContext, TimeSpan keepAliveInterval, ILoggerFactory loggerFactory, TimeSpan clientTimeoutInterval, int streamBufferCapacity)
```

Comment:
I think you need to keep the original constructor here to not introduce a breaking change.

Reply:
That’s right the breaks the SignalR service